telemetry-core/src/main/java/com/newrelic/telemetry/events/EventBatchSender.java

```diff
@@ -47,7 +48,7 @@
   public Response sendBatch(EventBatch batch) throws ResponseException {
     if (batch == null || batch.size() == 0) {
       logger.debug("Skipped sending of an empty event batch.");
-      return new Response(202, "Ignored", "Empty batch");
+      return EMPTY_BATCH_RESPONSE;
```

Comment:
`Reponse` is an immutable object composed of all immutable fields. No reason to create a new one every time this is called.

Reply:
Hey, nice catch!  (This is Jim)